addons/mail/static/tests/chatter_tests.js

```diff
@@ -520,6 +520,101 @@ QUnit.test('attachmentBox basic rendering', async function (assert) {
     form.destroy();
 });
 
+QUnit.test('attachmentBox drag and drop files [REQUIRE NON-INCOGNITO WINDOW]', async function (assert) {
```

Comment:
This test does too many things. The principle of unit tests is to have small tests, like:
- test there's a dropzone when hovering file
- test there's no dropzone after stop dragging
- test it uploads files on drag/drop